actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   `url_for` and `full_url_for` will now use "https://" as the default protocol
```

Comment:
Never heard of `full_url_for`. What's it used for? Doesn't seem like it's for apps to use in views/controllers.

Reply:
It was referring to [URL#full_url_for](https://github.com/rails/rails/blob/36b36fd57f5514e74d8b521c741d556a5a6a1251/actionpack/lib/action_dispatch/http/url.rb#L57).  I actually didn't realize [UrlFor#full_url_for](https://github.com/rails/rails/blob/820ffab6324161a1e5c9c045770a769e2232adb3/actionpack/lib/action_dispatch/routing/url_for.rb#L173) isn't public.  I have removed mention of it.